src/join/composer.jl

```diff
@@ -55,6 +56,42 @@ struct DataFrameJoiner
                                         "when matchmissing == :error"))
                 end
             end
+        elseif matchmissing === :notequal_copy
+            if kind in [:left, :semi, :anti]
```

Comment:
no need to allocate here, this is minor:
```suggestion
            if kind in (:left, :semi, :anti)
```